From d403c0bca48f2678c70763c3faca41c9adc76658 Mon Sep 17 00:00:00 2001
From: Johan Oudinet <johan.oudinet@gmail.com>
Date: Fri, 2 Aug 2024 17:39:07 +0200
Subject: [PATCH] fix ejabberdctl

Change default values so ejabberdctl run commands as ejabberd user.

Signed-off-by: Johan Oudinet <johan.oudinet@gmail.com>
Upstream: N/A (specific to buildroot)
---
 ejabberdctl.template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ejabberdctl.template b/ejabberdctl.template
index 83ec7e1bd..513c48c25 100755
--- a/ejabberdctl.template
+++ b/ejabberdctl.template
@@ -17,7 +17,7 @@ ERTS_VSN="{{erts_vsn}}"
 ERL="{{erl}}"
 EPMD="{{epmd}}"
 IEX="{{iexpath}}"
-INSTALLUSER="{{installuser}}"
+INSTALLUSER=ejabberd
 
 # check the proper system user is used
 case $(id -un) in
-- 
2.34.1

